### DISCUSSION TITLE
Does putting more cpu cores make the meta search engine go faster (taking advantage of the cores) or does it have the same performance (using only one core)?

### DISCUSSION THREAD

**Discussion by Klewltz** [searxng/searxng#3088] [Q&A] [answered]
I have seen that uwsgi only uses one core of my cpu, but when I change it to take advantage of all the cores of my processor I see that it has the same response speed (although it showed me that it could work with several cores in parallel).

**Comment by virtadpt** [comment, marked as answer]
The bottleneck isn't the cores sNG is running on top of, it's the other network services - the other search engines it's contacting.  If you look at your uwsgi workers, they're going to mostly be waiting for responses from outside.
